doc/installation.rst

```diff
@@ -0,0 +1,211 @@
+============
```

Comment:
_(non-blocking)_

Nice pandoc command in 1272b0ea00e5d90e87c7f14b6a6d3c6291685d8b. I like it better than what I had done in https://github.com/nextstrain/docs.nextstrain.org/commit/f97ae0c30d6ecb4377b321bbdc4d917563d424e7 and other previous conversions.